posture/serialnumber_ios.go

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+//go:build ios
```

Comment:
technically you shouldn't need this because `_ios.go` is already in the filename.

Reply:
done